_layouts/default.html

```diff
@@ -9,13 +9,15 @@
   <label id="color-mode-switch" for="color-mode">💡</label>
 
   <div class="site-container">
-
-    {% include header.html %}
-
-    {{ content }}
-
-    {% include footer.html %}
-
+    <div class="page-content">
+  
+      {% include header.html %}
+  
+      {{ content }}
+  
+      {% include footer.html %}
+  
+    <div>
```

Comment:
```diff
   <div class="site-container">
     <div class="page-content">
-  
+
       {% include header.html %}
-  
+
       {{ content }}
-  
+
       {% include footer.html %}
-  
+
     <div>
   <div>
```

Reply:
(or just drop the extra line breaks too)

Reply:
done!